changelogs/8.0.asciidoc

```diff
@@ -1,12 +1,66 @@
+
+https://github.com/elastic/apm-server/compare/v8.0.0-beta1\...v8.0.0-rc1[View commits]
+
+[float]
+==== Breaking Changes
+- Removed source map upload endpoint {pull}6447[6447]
+- Removed unsupported libbeat `processors` configuration {pull}6474[6474]
+- Removed `apm-server.aggregation.transactions.enabled` configuration option {pull}6495[6495]
+- Removed `apm-server.aggregation.service_destinations.enabled` configuration option {pull}6503[6503]
+- Removed legacy Jaeger gRPC/HTTP endpoints {pull}6417[6417]
+- Removed `apm-server.jaeger` configuration options {pull}6560[6560]
+- Removed `apm-server.instrumentation` configuration options in favor of `instrumentation` {pull}6560[6560]
+- Removed `apm-server.rum.{allowed_service,event_rate}` configuration option in favor of `apm-server.auth.anonymous.{allow_service,rate_limit}` {pull}6560[6560]
+- Removed `apm-server.{api_key,secret_token}` configuration options in favor of `apm-server.auth.{api_key,secret_token}` {pull}6560[6560]
+- Removed unused `span.start.us` field, and deprecated `span.http.*` fields {pull}6602[6602]
+- Removed `apm-server.register.ingest.pipeline` and `output.elasticsearch.pipeline` configuration options {pull}6575[6575]
+- Removed `logging.ecs` and `logging.json` config {pull}6613[6613]
+- Removed `apm-server.data_streams.enabled`, and `setup.*` configuration options {pull}6606[6606]
+- Removed `apm-server.sampling.keep_unsampled` configuration option; non-RUM unsampled transactions are always dropped {pull}6514[6514] {pull}6669[6669]
+- Removed interpretation of non-standard "component" attribute from OpenTelemetry spans; changed default span type and subtypes {pull}6834[6834]
+- Onboarding documents are no longer indexed {pull}6431[6431]
+
+[float]
+==== Bug fixes
+- `host.ip` is now stored as an array, as specified by ECS {pull}6694[6694]
+- In accord with ECS, the server logs now set `source.address` to the immediate network peer's IP address, and `client.ip` to the originating client IP if known {pull}6690[6690]
+- Agent configuration metrics are now sent to the metrics-apm.internal* data stream {pull}6767[6767]
+- Improve logged message when APM Server blocks ingestion to wait for the integration to be installed {pull}6796[6796]
+
+[float]
+==== Intake API Changes
+- `transaction.name` was added to the error objects in the intake API {pull}6539[6539]
+
+[float]
+==== Added
+- Introduced a delete phase for all data streams. Traces, errors and logs are kept for 10 days, metrics are kept for 90 days {pull}6480[6480]
+- Changed RUM traces to use a dedicated data stream (`traces-apm.rum`). RUM traces are kept for 90 days {pull}6480[6480]
+- Standalone apm-server can now fetch source maps uploaded to Kibana, when `apm-server.kibana` is configured {pull}6447[6447]
+- `apm-server` artifacts now have the apm java-attacher.jar packaged alongside them {pull}6593[6593]
+- Run the java attacher jar when configured and not in a cloud environment {pull}6617[6617]
```

Comment:
Do we want these included?

Reply:
Probably not, it likely won't be usable until at 8.2 -- let's mention it then.